nservicebus/messaging/forwarding.md

```diff
@@ -7,7 +7,18 @@ redirects:
 ---
 
 
-Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. If there is a need to collect all successfully processed messages from all endpoints, then use the [Auditing Feature](/nservicebus/operations/auditing.md) instead.
+Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. Forwarding messages is particularly useful in complex upgrade scenarios, when the old version and new version of a particular endpoint are running side-by-side.
+
+Note that forwarding is a solution to a specific problem and generally should be used for a limited transition period. In case of long-term need to duplicate messages the [Publish-Subscribe](/nservicebus/messaging/publish-subscribe/) pattern will be usually more appropriate.
+
+
+## Auditing vs Fowarding
+
+Technically [Auditing](/nservicebus/operations/auditing.md) and Forwarding are very similar, both send a copy of the processed message to another queue. The main difference are intended usage scenarios.
+
+Auditing is used for collecting information on what is happening in the system, therefore the audited message is enriched with additional [information regarding processing it](/nservicebus/operations/auditing.md#message-headers). The forwarded message is a copy of the message, without the additional auditing information.
+
+Note: In Version 5 and below the forwarded messages contain some of the headers added during auditing. Those headers are no longer added in Version 6 and above.
```

Comment:
Note: In Version 5 and below the forwarded messages contain some of the headers added during auditing. Those headers are no longer added in Version 6 and above.

How about:

NOTE: While some of the audit headers were available for the forwarded messages in Versions 5 and below, starting from Version 6, forwarded messages will no longer contain the audit message headers.